<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DateSettingForm</class>
 <widget class="QWidget" name="DateSettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="SliderSpin" name="spin1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>290</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>290</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SliderSpin" name="spin2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>290</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>290</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="SliderSpin" name="spin3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>290</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>290</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderSpin</class>
   <extends>QLabel</extends>
   <header>sliderspin.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
